.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: objective-c
+before_install:
+    - (ruby --version)
```

Comment:
Just prints out the version of ruby that tracks is using for debugging purposes :)